storage/api/Storage.Samples/MakePublic.cs

```diff
@@ -12,8 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// [START storage_make_public]
 // [START storage_set_bucket_public_iam]
 
+//TODO: remove the storage_set_bucket_public_iam tag later as it is already declared in MakeBucketPublic.cs
```

Comment:
Move this TODO to before both region tags so that it does not show on the sample itself in the docs. Everything in between the region tags will show on the sample.
(Sorry for not mentioning this before.)